test/honeysql_postgres/postgres_test.cljc

```diff
@@ -71,7 +71,7 @@
            (sql/format {:delete-from :distributors
                         :where [:> :did :10]
                         :returning [:*]})))
-    (is (= ["UPDATE distributors SET dname = ? WHERE did = 2 RETURNING did dname" "Foo Bar Designs"]
+    (is (= ["UPDATE distributors SET dname = ? WHERE did = 2 RETURNING did, dname" "Foo Bar Designs"]
```

Comment:
Actually, I think we should add some more entropy to this test, test for 1 item and 3 items respectively.